### PR TITLE
Added running with nginx proxy manager guide

### PR DESCRIPTION
**Pull request by TheDogHusky**
I added a running with nginx proxy manager without the internal docker caddy (so with docker) guide.
Feel free to review if I made any typos or anything, I tested it locally and it works fine and looks fine too.

### CODE REVIEW

File: packages/next/src/content/guides/running-with-nginx-proxy-manager.mdx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+You will need to setup Nginx Proxy Manager as well, on port 80, 443 and 81. Go to [#installing-nginx-proxy-manager](#installing-nginx-proxy-manager) for more information.
+
+You will also need a FQDN (Fully Qualified Domain Name) pointing to your server. You can use a service like [Cloudflare](https://www.cloudflare.com/) to get a domain.
+
+### Installing Nginx Proxy Manager
+
+First, you need to install Nginx Proxy Manager. You can follow the official guide [here](https://nginxproxymanager.com/guide/#quick-setup). But there are some little changes we need to make to the configuration.
+
+After you have created the docker-compose.yml file, you need to change the `image` field to `jc21/nginx-proxy-manager:github-pr-3478`. Why is so? Because the latest images has a bug in locations, which is fixed by this PR. Your docker-compose.yml file should look like this:
```
Comment (jamesbt365) [inline packages/next/src/content/guides/running-with-nginx-proxy-manager.mdx:23]: NginxProxyManager/nginx-proxy-manager#3478 may be closed but the most recent comments state that its fixed in the latest version.

Regardless of the case, using PRs as a base comes with additional risks that are not explicitly mentioned to the user here. Especially with a program that routes network traffic.

Comment (TheDogHusky) [reply packages/next/src/content/guides/running-with-nginx-proxy-manager.mdx:23]: Indeed. Seems like I didn't updated my NPM in the first place, thanks for the report, i'll work into a new commit.

File: packages/next/src/content/guides/running-with-nginx-proxy-manager.mdx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+You will also need a FQDN (Fully Qualified Domain Name) pointing to your server. You can use a service like [Cloudflare](https://www.cloudflare.com/) to get a domain.
+
+### Installing
+
+This guide assumes you will use Docker to run chibisafe. If you want to run it manually, just follow the [manual guide](/guides/running-manually) and install Nginx Proxy Manager like we said.
+
+First, move to a place where you can install it:
+
+```bash
+cd /home/<your user>
```
[the remark this answers is not in the document]
Comment (TheDogHusky) [reply packages/next/src/content/guides/running-with-nginx-proxy-manager.mdx:26]: I’ll hop into that tomorrow

[the remark this answers is not in the document]
Comment (TheDogHusky) [reply packages/next/src/content/guides/running-with-nginx-proxy-manager.mdx:26]: https://github.com/chibisafe/chibisafe/pull/666/commits/2bc86f31f25e8d4671a83dd616203f4e734a8e08

**Pull request merged**